DSCResources/cMsmqQueuePermissionEntry/cMsmqQueuePermissionEntry.psm1

```diff
@@ -363,4 +402,38 @@ function Reset-cMsmqQueueSecurity
     }
 }
 
+function Build-QueuePath()

```

Comment:
Please use an [approved verb](https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx) for this helper function.